components/swatch/stories/swatch.stories.js

```diff
@@ -22,33 +28,218 @@ export default {
 		},
 		rounding: {
 			name: "Rounding",
+			description: "The amount of corner rounding for a swatch.",
+			defaultValue: "regular",
 			type: { name: "string" },
 			table: {
 				type: { summary: "string", required: true },
 				category: "Component",
+				defaultValue: { summary: "Regular", },
 			},
 			options: ["none", "regular", "full"],
 			control: "select",
 		},
 		isDisabled,
 		isSelected,
+		borderStyle: {
+			name: "Border style",
+			type: { name: "string" },
+			table: {
+				type: { summary: "string", required: true },
+				category: "Component",
+			},
+			options: ["default", "noBorder", "lightBorder"],
```

Comment:
```suggestion
			options: ["default", "none", "light"],
```

What would you think of using a shorthand here for border and converting the classes in the `template.js`.